packages/fast-components-react-msft/app/components/examples.data.tsx

```diff
@@ -0,0 +1,8 @@
+import schema from "./react-html-element.schema.json";
+import ReactHTMLElement from "./react-html-element";
```

Comment:
Why are some things imported in Hungarian notation and others in Camel casing?  
`import ReactHTMLElement` but in app.tsx it's imported as `import reactHTMLElementExamples` ?  Should these be made consistent?  I think this is a problem in many places, but, we should start cleaning up if others agree this is an issue.

Reply:
I think this is likely due to ReactHTMLElement being a class vs reactHTMLElementExamples being an exported constant.

Reply:
Correct, classes are PascalCased, variables are camelCased